src/main/java/com/google/api/codegen/discovery/transformer/py/PythonSampleMethodToViewTransformer.java

```diff
@@ -0,0 +1,172 @@
+import com.google.protobuf.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PythonSampleMethodToViewTransformer implements SampleMethodToViewTransformer {
+
+  private static final String TEMPLATE_FILENAME = "py/sample.snip";
+
+  public PythonSampleMethodToViewTransformer() {}
+
+  @Override
+  public ViewModel transform(Method method, SampleConfig config) {
+    SampleTypeTable typeTable =
+        new SampleTypeTable(new PythonTypeTable(""), new PythonSampleTypeNameConverter());
+    SampleNamer namer = new PythonSampleNamer();
+    SampleTransformerContext context =
+        SampleTransformerContext.create(config, typeTable, namer, method.getName());
+    return createSampleView(context);
+  }
+
+  private SampleView createSampleView(SampleTransformerContext context) {
+    SampleConfig config = context.getSampleConfig();
+    MethodInfo methodInfo = config.methods().get(context.getMethodName());
+    SampleNamer namer = context.getSampleNamer();
+    SampleTypeTable typeTable = context.getSampleTypeTable();
+    SymbolTable symbolTable = SymbolTable.fromSeed(PythonTypeTable.RESERVED_IDENTIFIER_SET);
+
+    SampleView.Builder builder = SampleView.newBuilder();
+
+    String serviceVarName = symbolTable.getNewSymbol(namer.getServiceVarName(config.apiTypeName()));
+
+    if (methodInfo.isPageStreaming()) {
+      builder.pageStreaming(createSamplePageStreamingView(context, symbolTable));
+    }
+
+    // Created before the fields in-case there are naming conflicts in the symbol table.
+    SampleAuthView sampleAuthView = createSampleAuthView(context);
+
+    List<SampleFieldView> fields = new ArrayList<>();
+    List<String> methodParamAssigments = new ArrayList<>();
+    for (FieldInfo field : methodInfo.fields().values()) {
+      String name = namer.localVarName(Name.lowerCamel(field.name()));
+      SampleFieldView sampleFieldView =
+          SampleFieldView.newBuilder()
+              .name(name)
+              .defaultValue(typeTable.getZeroValueAndSaveNicknameFor(field.type()))
+              .example(field.example())
+              .description(field.description())
+              .build();
+      fields.add(sampleFieldView);
+      // Ex: "fooBar=foo_bar"
+      methodParamAssigments.add(field.name() + "=" + name);
+    }
+
+    boolean hasRequestBody = methodInfo.requestBodyType() != null;
+    if (hasRequestBody) {
+      String requestBodyVarName =
+          symbolTable.getNewSymbol(
+              namer.getRequestBodyVarName(methodInfo.requestBodyType().message().typeName()));
+      builder.requestBodyVarName(requestBodyVarName);
+      methodParamAssigments.add("body=" + requestBodyVarName);
+    }
+
+    boolean hasResponse = methodInfo.responseType() != null;
+    if (hasResponse) {
+      builder.responseVarName(symbolTable.getNewSymbol(namer.getResponseVarName()));
+    }
+
+    String credentialsVarName =
+        config.authType() == AuthType.API_KEY ? "developerKey" : "credentials";
```

Comment:
Just checked, this is the name that the client library uses, so in keeping with the convention where the arg and var names are the same I chose to call this var `developerKey` as well.

We had this slightly differently before:
```
service = discovery.build('translate', 'v2', developerKey='{MY-API-KEY}')
```

Now:
```
developerKey = ''  #TODO: Update placeholder with desired API key.

service = discovery.build('translate', 'v2', developerKey=developerKey)
```

I thought it was more consistent with the other samples